docs/concepts/gnovm.md

```diff
@@ -24,4 +24,59 @@ This enables non-web3 developers to contribute without requiring an understandin
 
 Install [`gno`](../getting-started/local-setup/local-setup.md) and refer to the [`examples`](https://github.com/gnolang/gno/tree/master/examples) folder to start developing contracts.
 
-Check the [Makefile](https://github.com/gnolang/gno/blob/master/gnovm/Makefile) to enhance GnoVM, Gno, and stdlibs.
+## Enhance or modify gnoVM
```

Comment:
I'm not sure why this section is called `Enhance or modify gnoVM`, when it discussed the Makefile in the `gnovm` directory. I'm not entirely convinced it should even be a section on this in the documentation, but just better comments in the Makefile